singlestoredb/fusion/handlers/sqrl.py

```diff
@@ -0,0 +1,50 @@
+import requests
+import os
+
+from typing import Any
+from typing import Dict
+from typing import Optional
+
+from singlestoredb.fusion.handler import SQLHandler
+from singlestoredb.fusion.result import FusionSQLResult
+
+class SqrlExplain(SQLHandler):
+    """
+    SQRL EXPLAIN string_value;
+
+    string_value = '<string-value>'
+
+    """
+    INKEEP_API_KEY = os.getenv('INKEEP_API_KEY')
+    INKEEP_INTEGRATION_ID = os.environ.get('INKEEP_INTEGRATION_ID')
+
+    def run(self, params: Dict[str, Any]) -> Optional[FusionSQLResult]:
+        prompt = "Explain me this SQL query:" + params['string_value']
+        apiUrl = "https://api.inkeep.com/v0/chat_sessions/chat_results"
```

Comment:
I think this should be a class variable as well so it can be changed easier for testing new API versions.